src/js/components/Box/stories/Background.js

```diff
@@ -59,12 +59,12 @@ export const BackgroundBox = () => (
     </Box>
     <Box
       background={{
-        color: { dark: 'darkgrey', light: 'lightgrey' },
```

Comment:
I still see a violation on this story

Reply:
I'm seeing an incomplete test but not a violation. Then when I run axe dev tools things look good there

Could you send the violation that's happening?